src/solver.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+const parser = require("./sudokuParser");
+const helper = require("./helper");
+
+function getRelatedCells(cell, sudokuMatrix) {
+    return new Array()
+        .concat(helper.extractRow(cell.row, sudokuMatrix))
+        .concat(helper.extractColumn(cell.col, sudokuMatrix))
+        .concat(helper.extractBlock(cell.row, cell.col, sudokuMatrix, 3));
+}
+
+function solve(sudokuString) {
+    let sudokuMatrix = parser.parseToArray(sudokuString);
+    const modifiableCells = helper.getEmptyCells(sudokuMatrix);
+    let nowModified = 0;
+
+    while (helper.inRange(0, modifiableCells.length - 1, nowModified)) {
+        let x = modifiableCells[nowModified].row;
+        let y = modifiableCells[nowModified].col;
+        if (sudokuMatrix[x][y] == null) sudokuMatrix[x][y] = 0;
```

Comment:
So you only set the cell to `0` so `helper.findUniqueInt` can work. You should move this if into helper function. 
Separate functions should not be aware of such details.